pkg/detector/library/bundler/advisory.go

```diff
@@ -17,8 +17,8 @@ type VulnSrc interface {
 
 // Advisory implements the bundler VulnSrc
 type Advisory struct {
-	vs       VulnSrc
 	comparer RubyGemsComparer
+	vs       VulnSrc
```

Comment:
I wonder if it's worth having this lint check. It seems a bit of an premature optimization to me and only is applicable in this case for x64 arch.

Reply:
I agree with you. We can disable the linter. It was easy to fix this time, so I've fixed it.